ocaml/idl/datamodel.ml

```diff
@@ -3889,7 +3889,9 @@ module VDI = struct
     ; {
```

Comment:
I see there are a few moe places that use `Datetime` in the datamodel, e.g. `datamodel_pool` has a field `telemetry_next_collection`, do we care about documenting those as well? Or do we just documenting ones that were not using Datetime before?